tests/typecheck/contrib/auth/test_decorators.yml

```diff
@@ -41,6 +41,28 @@
         from django.contrib.auth.decorators import user_passes_test
         @user_passes_test  # E: Argument 1 to "user_passes_test" has incompatible type "Callable[[HttpRequest], HttpResponse]"; expected "Callable[[Union[AbstractBaseUser, AnonymousUser]], bool]"
         def view_func(request: HttpRequest) -> HttpResponse: ...
+-   case: user_passes_test_has_user_of_type_auth_user_model
+    disable_cache: true
```

Comment:
Why would we need to disable cache?

Reply:
Oops, this is just a copy-past mistake 😇